kustomize/citadel/namespace.yaml

```diff
@@ -2,3 +2,5 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: istio-system
+  labels:
+    name: istio-system
```

Comment:
Why ?

Reply:
Otherwise our check for 
```
      - key: name
        operator: NotIn
        values:
        - {{ .Release.Namespace }}
```

Does not work

Reply:
`key:` only looks at labels

Reply:
I mean - why not adding injection=disabled.

Reply:
sgtm, should we just remove the name check then since it isn't documented to add `name` and we don't add name

Reply:
probably - but not a priority to remove it. 
I've seen other apps ( certmanager ) use the same pattern with foo-injection=disabled.